src/applications/hca/components/HCASubwayMap.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
```

Comment:
pulling this out into its own file since the component will be used by both the new IntroductionPage as well as the EnrollmentStatus component.